### DISCUSSION TITLE
Are there any current CU/DU split srsRAN deployments ?

### DISCUSSION THREAD

**Discussion by mrouili** [srsran/srsRAN_Project#433] [General Help] [answered]
Just wanted to confirm/clarify, currently all the srsRAN_Project deployment options (DPDK , k8s, 7.2 split..) have the CU/DU as one unit and it isn't possible to deploy the CU and DU separately on different machines or containers right ? 

Thanks in advance !

**Comment by P4sca1** [comment, marked as answer]
According to [the roadmap](https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html), CU / DU split is still work in progress.
